tasklist/app/controllers/tasks_controller.rb

```diff
@@ -7,4 +7,28 @@ def show
     id = params[:id]
     @task = Task.find(id)
   end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    Task.create(task_params[:task])
+
+  # OR
+  #
+  # t= Task.new(param[:task])
+  # t.save
+    redirect_to "/"
+  end
+
+  def destroy
+    Task.destroy(params[:id])
+    redirect_to "/tasks"
```

Comment:
I'm curious why you're redirecting to "/tasks" here and "/" above in the create method?